src/Microsoft.Extensions.Configuration.UserSecrets/UserSecretsIdAttribute.cs

```diff
@@ -6,15 +6,15 @@
 namespace Microsoft.Extensions.Configuration.UserSecrets
 {
     /// <summary>
-    /// Represents the user secrets id.
+    /// Represents the user secrets ID.
```

Comment:
Sounds good. BTW what is said above is just an attempt to express the idea we can convey. I trust you can come up with something better :smile: